### DISCUSSION TITLE
[Feature Request] Branding options

### DISCUSSION THREAD

**Discussion by NuttCorp** [docusealco/docuseal#16] [Ideas]
Be great to have branding options and ability to change up the main page, email templates and color scheme.

**Comment by AlexBTurchyn** [comment]
@NuttCorp Some of the above is in my road map, but these features will not appear in the near future. Sorry 😞

**Comment by 128keaton** [comment]
@AlexandrToorchyn would it just be possible to have the option replace the logo on signing pages?

**Comment by omohokcoj** [comment]
Branding and white label is a part of paid DocuSeal plans
